frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -84,26 +121,30 @@ export const MissionQueueCard = ({ order, mission, onDeleteMission }: MissionQue
         <StyledMissionCard key={mission.id} style={{ boxShadow: tokens.elevation.raised }}>
             <HorizontalContent>
                 <HorizontalNonButtonContent onClick={routeChange}>
-                    <CircularCard style={{ background: fillColor }}>
+                    <CircularCard id="item1" style={{ background: fillColor }}>
```

Comment:
Where are these ids used? What are they used for? If they are needed then we may at least want to give them a more generic name, since I think IDs are global scope in the document.

Reply:
I have changed the name, if you look up on the styles they are used ti adjust the grid with the screen size